openassessment/xblock/static/js/philu/lib/helpers.js

```diff
@@ -0,0 +1,4 @@
+$(document).on("keyup",".text-input textarea", function(){
+    var element = $(this);
```

Comment:
we can receive `event` in this callback and then we can use event.target directly instead making element from this and then using that element.

Reply:
we are not making elements here it is just to convert native dom element to jquery object.